text/0517-io-os-reform.md

```diff
@@ -1009,6 +1021,13 @@ cross-platform compatibility (in the case of `read_line`) and for ease
 in copying data without loss (in particular, distinguishing whether
 the last line included a final delimiter).
 
+`EINTR`-based errors are automatically restarted by `read_until`
+and `read_line`.  Short reads give a partial buffer and return the
```

Comment:
e.g. for `read_line`, it might not represent a line in the sense that it might not ever see a '\n'.

Or were you hinting that it was clear from discussion but not in the doc (and I should update the commit)?